### ISSUE TITLE
Some forms and menus have no standard hotkeys: Alt + <key>

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#34945]
### Description

For automation and users who likes to work with keyboard, it is usually useful to add  `Alt + <key>` hotkeys for all menu items and buttons of the most often used forms.
I mean these hotkeys with underscored symbol which appears on pressing and releasing Alt key.
Here in this window the only Close-button has Alt + C hotkey.
I suggest to add hotkeys for all these buttons as often used.

https://github.com/user-attachments/assets/905b8ae4-ce70-4ed4-aa04-6242cbd6a23f

Menu items should also have these hotkeys.
Notice the presense of underscore symbol in menu item names:

https://github.com/user-attachments/assets/e1e3e579-469a-4088-8986-83c34703f2e7

### DBeaver Version

Community Edition 24.1.3.202407122322

### Operating System

Windows 10

### Steps to reproduce

- pressing and releasing Alt key
- see items without underscored symbol in the name of menu and buttons

### Additional context

Please also check all other forms and menu.
Remember that ideally the chosen keys should be unique on the same nested level of menu items.

**Comment by ask9** [comment]
Don't forget this menu, its items are usually often used, but only "Generate" has a key, sub items also have no keys:

![изображение](https://github.com/user-attachments/assets/94aaad56-cc4a-41aa-a70a-fe962147f17a)

**Comment by arhayka** [comment]
Thank you for the idea!

**Comment by ask9** [comment]
![img](https://github.com/user-attachments/assets/7c37ebdb-8931-4b98-b611-3e51052dc499)

Suggestion about this issue.
Please try to choose different keys for `DDL ` and `DELETE ` commands.
Or maybe even any standalone hotkey for DDL (not `alt + key`), because `DDL` is used frequently.
